rust/macros/module.rs

```diff
@@ -215,9 +213,9 @@ struct ModuleInfo {
     type_: String,
     license: String,
     name: String,
-    author: Option<String>,
-    description: Option<String>,
-    alias: Option<String>,
+    author: String,
+    description: String,
+    alias: String,
```

Comment:
Not setting a property is different from setting the field to empty.

Reply:
yes, I believe this causes the build error also. I incorrectly assumed Default would resolve this. I'll fix it (tomorrow). Thanks for your review BTW.

Reply:
Removed this also via rebase.